hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/DeploymentRequest.java

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository.model;
+
+import org.eclipse.hawkbit.repository.exception.InvalidMaintenanceScheduleException;
+import org.eclipse.hawkbit.repository.model.Action.ActionType;
+
+/**
+ * A custom view on assigning a {@link DistributionSet} to a {@link Target}.
+ *
+ */
+public class DeploymentRequest {
```

Comment:
Is there sth like a DeploymentRequestBuilder to create a DeploymentRequest based on a static builder pattern? The DeploymentManagement interface could offer a method "deploymentRequest() which returns a DeploymentRequestBuilder object...

Not sure how often DeploymentRequest objects need to be created... depending on that a builder pattern might be helpful...